bindings/dotnetcore/dotnet-core-binding/Microsoft.Azure.Devices.Gateway/GatewayInterop.cs

```diff
@@ -29,6 +55,39 @@ public static IntPtr CreateFromJson(string file_path)
             return CreateFromJsonInternal(file_path);
         }
 
+        /// <summary>
+        ///   Update the gateway with module, link details etc.
+        /// <param name="gw">#GATEWAY_HANDLE to be started</param>
+        /// <param name="file_path">Path to the JSON configuration file to upate from .</param>
+        /// </summary>
+        public static int UpdateFromJson(IntPtr gw, string file_path)
+        {
+            var json_string = File.ReadAllText(file_path);
```

Comment:
/nit:reSharper Warning - Can you put explicit variable type. 
We usually use var when the type is clear. On this case we would have to check that is the return of ReadAllText.